### DISCUSSION TITLE
Does mantine support markdown format

### DISCUSSION THREAD

**Discussion by dibdutta** [mantinedev/mantine#2798] [Feature requests]
My requirement is render markdown to html. I tried `RichTextEditor` from `"@mantine/rte"` . But it expects an `html` element , does not work with `markdown` .

Also if `mantine` doesn't support `markdown` . I shall able to use some 3rd party module to convert `markdown` to `html`. But apart from `RichTextEditor`  is there a way to render `html` string in `mantine`. For e.g. 
`<Text >{converter.makeHtml("# hello, markdown!")}</Text>`
renders below in `ui`
`<h1 id="hellomarkdown">hello, markdown!</h1>`

**Comment by rtivital** [comment]
You can render HTML with this component – https://mantine.dev/core/typography-styles-provider/

**Reply by snturk** [reply]
Expired link, here's the updated one -> https://mantine.dev/core/typography/
